### DISCUSSION TITLE
PrimeIcons not working

### DISCUSSION THREAD

**Discussion by changelahiren** [primefaces/community#1655] [PrimeIcons]
Hi Team,
I hope you are doing well!
Suddenly, PrimeIcons not working. Below CDN for Prime Icons not working. Please refer to the below screenshots for the same.

https://unpkg.com/primeicons/primeicons.css

We have used the below node modules for PrimeReact, PrimeIcons, and Prime Flex.

"primeflex": "^3.3.1",
"primeicons": "^6.0.1",
"primereact": "^9.6.1",

Please help us to resolve this prime icons issue. When can we expect the above CDN for PrimeIcons to be available?
![Screenshot 2024-04-12 134607](https://github.com/primefaces/community/assets/94886023/d0d7f30b-530a-4be9-b366-07091a5363ac)
![Screenshot 2024-04-12 134848](https://github.com/primefaces/community/assets/94886023/79a75f17-7e92-43be-9375-eda59d9a6a3e)

**Comment by onursenture** [comment]
Hi, Did you try https://unpkg.com/primeicons@7.0.0/primeicons.css? The link you shared is redirecting to the 7.0.0 URL.
